doc/Hammer-Use/Flow-Control.rst

```diff
@@ -0,0 +1,34 @@
+.. _flow-control:
+
+Flow Control
+=======================================
+
+Physical design is necessarily an iterative process, and designers will often require fine control of the flow within any given action.
+This allows for rapid testing of new changes in the Hammer IR  to improve the quality of results after certain steps.
+
+Given the flow defined by tool steps and hooks (described in the next section), users can select ranges of these to run.
+The tool script will then be generated with commands corresponding to only the steps that are to be run.
+
+Command-line Interface
+----------------------
+
+Flow control is specified with the following optional Hammer command-line flags, which must always target a valid step/hook:
+
+* ``--from_step <target>``: this starts the tool from (inclusive) the target step
+* ``--after_step <target>``: this starts the tool from (exclusive) the target step
```

Comment:
Should this be maybe `start_after_step`, `post_step`, or `resume_after`? `after step` is ambiguous in that it's not clear whether it means to stop after or start after.

Reply:
I don't see how `after_step` could be interpreted as stopping after, but is this documentation not clear about which flag is for what?

If we want to completely disambiguate it, we should use `resume_pre_step`, `resume_post_step`, `pause_pre_step`, and `pause_post_step` (could replace resume with start and pause with stop).

Reply:
Just looking at it directly it can imply `[to-]after-step`

Reply:
Recording an offline discussion with @colinschmidt :

What about having `--{start|stop}_{before|after}_step` + `--only_step` (which is synonymous with `--start_before_step x --stop_after_step x` as the documented 5 steps, and then leave behind the old names (`--to_step`, `--from_step`, etc.) as undocumented aliases for these core 5?